src/openpgp.js

```diff
@@ -540,15 +541,15 @@ export async function encryptSessionKey({ data, algorithm, aeadAlgorithm, encryp
 }
 
 /**
- * Decrypt symmetric session keys with a private key or password. Either a private key or
+ * Decrypt symmetric session keys with private keys or passwords (not both). Either a private key or
  *   a password must be specified.
```

Comment:
```suggestion
 * Decrypt symmetric session keys with a private key or password (not both). Either private keys or
 *   passwords must be specified (passing both is not supported).
```
The distinction is somewhat subtle, but it makes slightly more sense to me this way because while you can pass multiple private keys or passwords, it ends up (succesfully) decrypting the message using (at most) only one of them.

Reply:
Uhm, I'd leave the singular everywhere then, because we still require only 1 key or password (and we have the singular notation in the `encrypt*` functions)

Reply:
I think "Either private keys or passwords must be specified" could be taken to refer to the option names `privateKeys` and `passwords` - though actually now that it's `decryptionKeys` it should probably say that instead. I don't think it necessarily implies that you must pass more than one, since neither does the option name `decryptionKeys`. But yeah, we should probably be a bit consistent across all functions.